app/models/phone_configuration.rb

```diff
@@ -9,14 +9,11 @@ class PhoneConfiguration < ApplicationRecord
   enum delivery_preference: { sms: 0, voice: 1 }
 
   def formatted_phone
-    Phonelib.parse(phone).international
+    PhoneFormatter.format(phone)
```

Comment:
It took me some digging, but I'm confident that these two are equivalent; `PhoneFormatter.format` sends `'US'` as the country_code, which the code you're replacing does not, but we set Phonelib's default country to `'US'` in `config/initializers/phonelib.rb`.